amr-wind/core/FieldBCOps.H

```diff
@@ -11,6 +11,17 @@
 
 namespace amr_wind {
 
+namespace {
+int CC_idx(int /* idir */)
```

Comment:
I wonder if for some of these we can use stuff in: https://github.com/Exawind/amr-wind/blob/main/amr-wind/utilities/DirectionSelector.H#L56. Kinda like we do for `IndexSelector`

Reply:
took some inspiration from there.